### PR TITLE
Unrecommend using menus and conversations together

### PR DESCRIPTION
**Pull request by KnorpelSenf**
https://t.me/grammyjs/145782

### CODE REVIEW

**Review by roj1512 — APPROVED** [review]

**Comment by MajorLettuce** [comment]
👍

**Review by deptyped — APPROVED** [review]

**Review by habemuscode — APPROVED** [review]

File: site/docs/es/plugins/conversations.md [hunk 1]
```diff
@@ -989,9 +989,35 @@ Como se mencionó [anteriormente](#introduccion), los manejadores grammY siempre
...
+:::warning Menús interactivos dentro de conversaciones
+Con el [plugin de menú](./menu.md), estos conceptos chocan mucho.
+Aunque los menús _pueden_ funcionar dentro de las conversaciones, no recomendamos usar estos dos plugins juntos.
+En su lugar, utilice el plugin [inline keyboard plugin](./keyboard.md#teclados-en-linea) (hasta que añadamos soporte nativo de menús para conversaciones).
+Puedes esperar consultas específicas usando `await conversation.waitForCallbackQuery("my-query")` o cualquier consulta usando `await conversation.waitFor("callback_query")`.
+
+```ts
+const keyboard = new InlineKeyboard()
+  .text("A", "a").text("B", "b");
+await ctx.reply("A or B?", { reply_markup: keyboard });
```
Comment (CikiMomogi) [inline site/docs/es/plugins/conversations.md:1002]:
```suggestion
await ctx.reply("¿A o B?", { reply_markup: keyboard });
```
cmmiw @habemuscode

File: site/docs/zh/plugins/conversations.md [hunk 2]
```diff
@@ -989,11 +989,35 @@ async function waitForMe(conversation, ctx) {
...
+
+使用 [menu 插件](./menu.md)，这些概念会产生很严重的冲突。
+虽然菜单_可以_在对话中使用，但我们不建议同时使用这两个插件。
+取而代之地，使用常规的 [inline keyboard 插件](./keyboard.md#inline-keyboards)（直到我们为对话添加原生菜单支持）。
+你可以使用 `await conversation.waitForCallbackQuery("my-query")` 等待特定的回调查询，或者使用 `await conversation.waitFor("callback_query")` 等待任何查询。
+
+```ts
+const keyboard = new InlineKeyboard()
+  .text("A", "a").text("B", "b");
+await ctx.reply("A or B?", { reply_markup: keyboard });
```
Comment (CikiMomogi) [inline site/docs/zh/plugins/conversations.md:1004]:
```suggestion
await ctx.reply("A还是B？", { reply_markup: keyboard });
```
cmmiw

File: site/docs/zh/plugins/conversations.md [hunk 3]
```diff
@@ -989,11 +989,35 @@ async function waitForMe(conversation, ctx) {
...
+const response = await conversation.waitForCallbackQuery(["a", "b"], {
+  otherwise: (ctx) => ctx.reply("Use the buttons!", { reply_markup: keyboard }),
```
Comment (CikiMomogi) [inline site/docs/zh/plugins/conversations.md:1006]:
```suggestion
  otherwise: (ctx) => ctx.reply("使用这些按钮!", { reply_markup: keyboard }),
```
cmmiw

**Review by LWJerri — APPROVED** [review]

File: site/docs/es/plugins/conversations.md [hunk 4]
```diff
@@ -989,9 +989,35 @@ Como se mencionó [anteriormente](#introduccion), los manejadores grammY siempre
 Sin embargo, con las conversaciones, puedes procesar muchas actualizaciones en secuencia como si todas estuvieran disponibles al mismo tiempo.
 Los plugins hacen esto posible almacenando objetos de contexto antiguos, y reabasteciéndolos más tarde.
 Esta es la razón por la que los objetos de contexto dentro de las conversaciones no siempre se ven afectados por algunos plugins de grammY de la manera que cabría esperar.
+
+:::warning Menús interactivos dentro de conversaciones
```
Comment (roj1512) [inline site/docs/es/plugins/conversations.md:993]:
```suggestion
:::warning Menús interactivos dentro de conversaciones

```

**Review by roj1512 — APPROVED** [review]

**Pull request merged**